editor/plugins/script_text_editor.cpp

```diff
@@ -977,6 +977,21 @@ void ScriptTextEditor::_lookup_symbol(const String &p_symbol, int p_row, int p_c
 				emit_signal("go_to_help", "class_global:" + result.class_name + ":" + result.class_member);
 			} break;
 		}
+
+	}
+	// Every symbol other than absolute path is relative path so keep this condition at last
+	else if (p_symbol.is_rel_path()) {
```

Comment:
`else` should be on the same line as the closing bracket. You can put the comment inside the block (and end it with a dot).

The first empty line is also not needed.